README-yearnBorg.md

```diff
@@ -58,25 +58,35 @@ graph TD
     class yearnDaoVoting yearn;
 ```
 
+## Initial BORGing of ychad
+
+To implement the BORG, ychad unilaterally: 
+- determines initial signer set (i.e., keep existing signers)
+- approves/adopts legal agreements (Cayman Foundation)
+- installs SAFE modules (BORG implants) and guard (BORG core)
+
+If desired, can seek prior DAO social approval for these changes (and this is likely best for legitimacy), but no DAO onchain actions or legal actions are required. 
+
 ## Restricted Admin Operations
 
-`ychad.eth` will be prohibited from unilaterally performing the following admin operations:
+Once ychad is "BORGed", the following actions will require bilateral approval of the DAO and ychad. Onchain, this means 'blacklisting' certain unilateral SAFE operations that would otherwise be possible, instead requiring DAO/ychad co-approval of such actions:
 
 - Add / remove / swap signers / change threshold
 - Add / disable Modules
 - Set Guards
 
 ### Co-approval Workflows
 
-Except existing signers, Modules (BORG Implants), Guard (BORG Core) and its set rules,
-all coming operations listed above will require approval of both `ychad.eth` and DAO, with a process as such:
+The process for bilateral `ychad.eth` / DAO approvals will be as follows:
 
-1. Operation is initiated on the MetaLeX OS webapp
+1. Operation is initiated on the MetaLeX OS webapp [can snapshot's UI also be used as a fallback option?]
 2. A Snapshot proposal will be submitted via API using Yearn's existing voting settings
-3. MetaLeX's Snapshot oracle (`oracle`) will submit the results on-chain to an executor contract (`Snapshot Executor`), which will have the proposed transaction pending for co-approval
+3. MetaLeX's Snapshot oracle (`oracle`) will submit the results on-chain to an executor contract (`Snapshot Executor`), which will have the proposed transaction pending for co-approval [let's discuss fallback options if our oracle were to go offline, let's discuss security measures around oracle]
```

Comment:
- Will add features (heartbeat/TTL) to address oracle failover
- Will review oracle security